### PR TITLE
Improve Configuration Management standard

### PR DESCRIPTION
**Pull request by boffbowsh**
A few changes:
- Differentiates between managment of servers and underlying infrastructure
- Removes confusion of AMIs, the previous paragraph describing this didn't make sense
- Reverts to the intention of the original author (9e498147c05db6a8b4a2535572bd838cb6743fc6)
- Link to a solution-agnositic definition of configuration management
- Specify the code must be stored in Git

### CODE REVIEW

File: source/standards/configuration-management.html.md.erb [hunk 1]
```diff
@@ -7,18 +7,22 @@ expires: 2018-06-30
...
+Use [configuration management][] to manage, automate and standardise your infrastructure. When using configuration management you store your [infrastructure as code][] and put that code in a version control system such as Git.
 
-A particularly good use case for configuration management is virtual machines, for example, when you run a VM in [Amazon Elastic Compute Cloud (Amazon EC2)][] or as an instance in an [Amazon Machine Image (AMI)][].This will help you track and control changes in large software deployments. You will also automatically be building environments consistently - all new environments produced from the same configuration will be identical.
+We recommend using [Puppet][] to configure servers and virtual machines. Puppet is suitable for most systems including less complex ones, is scalable and widely used within GDS. If your environment consists of a simple deployment artefact like an [Amazon Machine Image (AMI)][], Puppet may not be necessary, but you must still set out and version control the artefact's build process.
```
Comment (soupdragon99) [inline source/standards/configuration-management.html.md.erb]: How about...

 'you must still put into code and version control the artefact's build process.'

Comment (boffbowsh) [reply source/standards/configuration-management.html.md.erb]: Sure

**Comment by boffbowsh** [comment]
I've squashed your commits. Happy to merge this once you've approved it.

**Comment by boffbowsh** [comment]
@soupdragon99 can you approve this once you're happy please?

**Pull request merged**